### DISCUSSION TITLE
Tags

### DISCUSSION THREAD

**Discussion by atomczak** [buildingSMART/IDS#322] [Ideas]
I have a long IDS file with many aspects and would like to group them, for example, into acoustic, structural and fire-related. This could be solved by adding a Tag or Label attribute on the Specification object. Right now, the IDS 1.0 already have Name, Description, Identifier and Instructions, none of which really supports my use case.

**Comment by NickNisbet** [comment]
Grouping of clauses would be supported by allowing Specifications to include Specifications as Facets. Then it would be clear if each was a Requirement or some were alternative Selections or Exceptions or whatever.

**Reply by atomczak** [reply]
I think yours is a valid but different use case. I have different requirements for different objects (so different Specifications) but want to be able to easily filter them by associated tag. Right now this is usually addressed by creating separate IDS files. Can you @NickNisbet post your idea about Sub-clauses and Exceptions (separately) in the Discussions/Ideas? I was about to do it, but even better if it comes from you. You can copy-paste what you wrote in the Issues earlier today.
